STX.REST.RESTFulSense.Clients/Brokers/Https/HttpBroker.cs

```diff
@@ -0,0 +1,35 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace STX.REST.RESTFulSense.Clients.Brokers.Https
+{
+    internal partial class HttpBroker : HttpClient, IHttpBroker
```

Comment:
Why inherits from HttpClient?
Do you use inheritance and composition at the same time?

Reply:
As there was or isn't a detailed blueprint to go by as of yet for this endeavor of re-writing the library this was a running start to get the ball bouncing I guess. Based on the repository project statement of "A client / server library wrapper around the existing .NET Core `HttpClient` implementation to provide meaningful exceptions for APIs response status codes." I went with it. Pr is draft now, and should have been so if you want to pull the branch and offer some input that would be great 👍

Reply:
I think the HttpClient inheritance could be removed, because an HttpClient is constructed via the HttpBroker constructor. :)